perun-core/src/main/java/cz/metacentrum/perun/core/impl/GroupsManagerImpl.java

```diff
@@ -279,16 +281,17 @@ public boolean isUserMemberOfGroup(PerunSession sess, User user, Group group) th
 
 	public List<Member> getGroupMembers(PerunSession sess, Group group) throws InternalErrorException {
 		try {
-			return jdbc.query("select " + MembersManagerImpl.memberMappingSelectQuery + ", groups_members.membership_type as membership_type from groups_members join members on members.id=groups_members.member_id " +
-					" where groups_members.group_id=?", MembersManagerImpl.MEMBER_MAPPER, group.getId());
+			return jdbc.query("select " + MembersManagerImpl.memberMappingSelectQuery + ", groups_members.membership_type as membership_type,  " +
+					"groups_members.source_group_id as source_group_id from groups_members join members on members.id=groups_members.member_id " +
```

Comment:
Guess the MembersManagerImpl.memberMappingSelectQuery could be expanded with ", groups_members.membership_type as membership_type, ..."